docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,989 @@
+---
+title: AWS Architecture and Resources
+---
+
+# AWS Architecture and Resources
+
+In this section we will talk about the AWS Resources and Openshift objects generated by Hypershift and how the Controllers interact with them during a Hosted Cluster deployment.
```

Comment:
As I'm reading through this, I see several references to HyperShift. I believe the official name is hosted control planes. Is there a guideline about when "HyperShift" is correct and when "hosted control planes" is correct?

Reply:
Hey @lahinson Thank you for the comment, we decided to move this `index.md` file to other PR in order to push the priority in the HOSTEDCP-325. I will reference this comment in the new PR in order to continue with he conversation.